tests/roles/cinder_adoption/tasks/main.yaml

```diff
@@ -97,6 +97,13 @@
   retries: 5
   delay: 2
 
+- name: Clean up old cinder-backup and cinder-scheduler services
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc exec -t cinder-scheduler-0 -- cinder-manage service remove cinder-backup standalone.localdomain
+    oc exec -t cinder-scheduler-0 -- cinder-manage service remove cinder-scheduler standalone.localdomain
```

Comment:
Not every CI job is running on Standalone, this would likely fail periodic jobs which have multi-controller configurations. If we want to clean up the services we'll need an approach that detects the host(s) where they ran.

Also we should be editing docs and tests in tandem, otherwise this applies only to our testing and not to user's environments.

Reply:
I've updated the logic in another PR, can you take a look? https://github.com/openstack-k8s-operators/data-plane-adoption/pull/594

The docs already have the procedure but it was missing from the cinder tasks.
Can you please elaborate on "tandem", not sure what it means.